osu.Game/Migrations/20181007180454_StandardizePaths.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System.IO;
+
+namespace osu.Game.Migrations
+{
+    public partial class StandardizePaths : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            string sanitized = Path.DirectorySeparatorChar.ToString();
+            string standardized = "/";
+
+            // If we are not on Windows, no changes are needed.
+            if (string.Equals(standardized, sanitized, StringComparison.Ordinal))
+                return;
+
+            // Escaping \ does not seem to be needed.
+            migrationBuilder.Sql($"UPDATE `BeatmapInfo` SET `Path` = REPLACE(`Path`, '{sanitized}', '{standardized}')");
+            migrationBuilder.Sql($"UPDATE `BeatmapMetadata` SET `AudioFile` = REPLACE(`AudioFile`, '{sanitized}', '{standardized}')");
+            migrationBuilder.Sql($"UPDATE `BeatmapMetadata` SET `BackgroundFile` = REPLACE(`BackgroundFile`, '{sanitized}', '{standardized}')");
+            migrationBuilder.Sql($"UPDATE `BeatmapSetFileInfo` SET `Filename` = REPLACE(`Filename`, '{sanitized}', '{standardized}')");
+            migrationBuilder.Sql($"UPDATE `SkinFileInfo` SET `Filename` = REPLACE(`Filename`, '{sanitized}', '{standardized}')");
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            string sanitized = Path.DirectorySeparatorChar.ToString();
+            string standardized = "/";
+
+            // If we are not on Windows, no changes are needed.
+            if (string.Equals(standardized, sanitized, StringComparison.Ordinal))
+                return;
+
+            migrationBuilder.Sql($"UPDATE `BeatmapInfo` SET `Path` = REPLACE(`Path`, '{standardized}', '{sanitized}')");
+            migrationBuilder.Sql($"UPDATE `BeatmapMetadata` SET `AudioFile` = REPLACE(`AudioFile`, '{standardized}', '{sanitized}')");
```

Comment:
i'm pretty sure these `BeatmapMetadata` ones are not relevant to this change?

Reply:
You're right, they are still not standardized so backgrounds or audio files that are not in the root directory would still break. I will fix that when I get a chance.

Reply:
AudioFile and BackgroundImage are now standardized too (they weren't before), and I doublechecked that paths in storyboards are also standardized.